src/clarity-angular/wizard/wizard.ts

```diff
@@ -124,6 +126,8 @@ export class Wizard implements OnInit, OnDestroy, AfterContentInit, DoCheck {
     @Output("clrWizardOnPrevious") onMovePrevious: EventEmitter<any> =
         new EventEmitter<any>(false);
 
+    @Input("clrWizardPreventDefaultNext") stopNext: boolean = false;
+
     @Input("clrWizardPreventDefaultCancel") stopCancel: boolean = false;
 
     @Input("clrWizardPreventModalAnimation") _stopModalAnimations: boolean = false;
```

Comment:
stopModalAnimations is given an underscore to denote that it should only be used internally (privately). The wizard needs it to make the inline/static wizard. It's how it tells the modal not to run animations.